src/components/telemetry_monitor/include/telemetry_monitor/telemetry_monitor.h

```diff
@@ -88,7 +92,7 @@ class TelemetryMonitor {
   virtual void Stop();
   virtual void Start();
   virtual void SendMetric(utils::SharedPtr<MetricWrapper> metric);
-  void set_streamer(Streamer* streamer);
```

Comment:
@dev-gh This signature needs to be deprecated, it cannot be deleted until the next major release (5.0.0)

Reply:
@jacobkeeler Done 2b29c82396c6b24e15f8d54405821f39de379741